lib/Config.cpp

```diff
@@ -53,6 +60,27 @@ void LibConfig::Config::sendJson(const json& data)
   requestStr.copy(buff, requestStr.size());
   socket->write(buff, requestStr.size());
   delete buff;
+  socketLoop->run<uvw::Loop::Mode::ONCE>();
+}
+
+///
+/// \todo get response and set-up name and id
+///
+void LibConfig::Config::loadConfig(Key const &givenKey)
+{
+  json request;
+  request["REQUEST_NAME"] = "CONFIG_LOAD";
+
+  if (givenKey.type == LibConfig::READ_WRITE) {
```

Comment:
I'd use a switch and assert on the default case (safer, more readable). It might cost a little performance but I think we can afford it.